lib/adaptors/serialport.js

```diff
@@ -119,3 +119,7 @@ Adaptor.prototype.onRead = function read(callback) {
 Adaptor.prototype.close = function close(callback) {
   this.serialport.close(callback);
 };
+
+Adaptor.prototype.newSerialport = function() {
+  return new SerialPort(this.conn, {});
+};
```

Comment:
We really don't need this level of abstraction. Let the test coverage score suffer a little bit, I'd rather have that than meaningless redirects in the code.